testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -664,7 +664,7 @@ impl BitcoinRegtestController {
 
         // RBF
         let tx_fee = self.config.burnchain.burnchain_op_tx_fee
-            + (attempt * self.last_tx_len * self.min_relay_fee);
+            + ((attempt - 1) * self.last_tx_len * self.min_relay_fee / 1024);
```

Comment:
For good measure, can you change `attempt - 1` to `attempt.saturating_sub(1)` so it doesn't ever underflow?

Reply:
Also, can you group the numerator explicitly?  I.e. `((attempt.saturating_sub(1) * self.last_tx_len * self.min_relay_fee) / 1024);`

Reply:
good idea.  addressed in 00d7880